vector/src/main/java/im/vector/app/features/spaces/leave/SpaceLeaveAdvancedFragment.kt

```diff
@@ -72,10 +139,38 @@ class SpaceLeaveAdvancedFragment @Inject constructor(
 
     override fun invalidate() = withState(viewModel) { state ->
         super.invalidate()
+
+        if (state.isFilteringEnabled) {
+            views.appBarLayout.setExpanded(false)
+        }
+
+        val behavior = (views.appBarLayout.layoutParams as? CoordinatorLayout.LayoutParams)?.behavior as? ToggleableAppBarLayoutBehavior
+        behavior?.let {
+            it.isEnabled = !state.isFilteringEnabled
+        }
+
+        updateRadioButtons()
+
         controller.setData(state)
     }
 
     override fun onItemSelected(roomSummary: RoomSummary) {
         viewModel.handle(SpaceLeaveAdvanceViewAction.ToggleSelection(roomSummary.roomId))
     }
+
+    private fun updateRadioButtons() = withState(viewModel) { state ->
+        (state.allChildren as? Success)?.invoke()?.size?.let { allChildrenCount ->
+            when (state.selectedRooms.size) {
+                0                -> {
+                    views.spaceLeaveSelectNone.isChecked = true
```

Comment:
Since they are in radio group it's not really necessary. I'd like to check them through the group btw, but this cause extra check callback to come and mess up entire thing. This entire radiogroup is most likely a temporary solution, since there is an intent to rework ui of screens where lists of checkboxes are presented to have a "one checkbox to rule them all" on top of screen as by material design guidelines

Reply:
thanks for the context 👍